unidist/core/backends/mpi/core/communication.py

```diff
@@ -324,14 +324,14 @@ def _send_complex_data_impl(comm, s_data, raw_buffers, len_buffers, dest_rank):
         Serialized data as bytearray.
     raw_buffers : list
         Pickle buffers list, out-of-band data collected with pickle 5 protocol.
-    len_buffers : list
-        Size of each buffer from `raw_buffers` list.
+    buffer_count : list
+        A `ComplexDataSerializer.buffer_count` entity required for deserialization.
```

Comment:
```suggestion
        List of the number of buffers for each object
        to be serialized/deserialized using the pickle 5 protocol.
        See details in :py:class:`~unidist.core.backends.mpi.core.serialization.ComplexDataSerializer`.
```
Fix evewhere other than in `unidist/core/backends/mpi/core/serialization.py`.